tools/parameter_tuning.py

```diff
@@ -0,0 +1,47 @@
+from sisyphus import Job, Task, tk
+from typing import Any, List, Tuple
+
+import numpy as np
+
+from i6_core.util import instanciate_delayed
+
+
+class PickOptimalParametersJob(Job):
+    """
+    Pick a set of optimal pickleable parameters based on their assigned (dynamic) score value.
+    """
+
+    def __init__(self, parameters: List[Tuple[Any]], values: List[tk.Variable], mode="minimize"):
```

Comment:
I would also remove the default for `mode`. It's very arbitrary to have "minimum" being the default here.